sdk/modelsrepository/Azure.IoT.ModelsRepository/samples/ModelsRepositoryClientSamples/ModelsRepositoryClientSamples.csproj

```
@@ -4,8 +4,11 @@
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>Azure.IoT.ModelsRepository.Samples</RootNamespace>
   </PropertyGroup>
+
+  <!-- NU5104: Using a project reference to the client causes build warning/errors on build agents. -->
+  <!-- CS8002: The non-track2 package dependency (Microsoft.Azure.DigitalTwins.Parser) is not signed. -->
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|AnyCPU' Or '$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <NoWarn>CS8002</NoWarn>
+    <NoWarn>CS8002;NU5104</NoWarn>
```

```suggestion
    <NoWarn>$(NoWarn);CS8002;NU5104</NoWarn>
```
I also recommend removing the condition.